cogs/assistance-cmds/vccompatlists.wiiu.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Wii U inject Compatibility list sites
+help-desc: Lists various links to Wii U VC compatibility sites
+aliases: compatlists
+---
+
+You can see what games are compatibile with the Wii U/WiiVC from these sites.
+
+Wii:
+https://wiki.gbatemp.net/wiki/WiiU_VC_Wii_inject_compatibility_list
```

Comment:
The compat lists (minus gamecube) should use the UWUVCI ones. who knows how out of date the gbatemp ones are..